faunadb/functions_auth.go

```diff
@@ -67,23 +67,18 @@ type identifyFn struct {
 // Returns:
 //  Ref - The reference associated with the current key.
 //
+// Deprecated: Use CurrentIdentity instead
+//
+//
 // See: https://app.fauna.com/documentation/reference/queryapi#authentication
-func Identity() Expr { return identityFn{Identity: NullV{}} }
-
-type identityFn struct {
-	fnApply
-	Identity Expr `json:"identity" faunarepr:"noargs"`
-}
+func Identity() Expr { return CurrentIdentity() }
```

Comment:
I think it's best to keep the old version otherwise there will be a disparity between the driver and the database API.

Reply:
So I discard this PR ?

Reply:
I personally don't think it's necessary. Keeping parity between drivers and the database API makes it easier to reason about their deprecation cycles.

Reply:
Agree. Closing this one.